### PR TITLE
[Fix] Fix missing package directory error

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Related Issue: #71 

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mim/commands/list.py [hunk 1]
```diff
@@ -52,7 +52,20 @@ def list_package(all: bool = False) -> List[Tuple[str, ...]]:
             if not home_page:
                 home_page = pkg.location
 
-            installed_path = get_installed_path(pkg_name)
+            if pkg_name.startswith('mmcv'):
+                pkgs_info.append((pkg_name, pkg.version, home_page))
+                continue
+
+            try:
+                # some packages can not obtained installed_path like
```
Comment (ZwwWayne) [inline mim/commands/list.py:60]: fix grammar issues like `can not obtained`

File: mim/commands/list.py [hunk 2]
```diff
@@ -52,7 +52,20 @@ def list_package(all: bool = False) -> List[Tuple[str, ...]]:
...
+                # threadpoolctl. we can ignore those packages because
+                # `mim list` just lists those packages they have package
```
Comment (ZwwWayne) [inline mim/commands/list.py:62]: those packages that

File: mim/commands/list.py [hunk 3]
```diff
@@ -52,7 +52,20 @@ def list_package(all: bool = False) -> List[Tuple[str, ...]]:
...
 
-            installed_path = get_installed_path(pkg_name)
+            if pkg_name.startswith('mmcv'):
+                pkgs_info.append((pkg_name, pkg.version, home_page))
+                continue
+
+            try:
+                # `installed_path` of some packages can not be obtained like
+                # threadpoolctl. We can ignore those packages because
+                # `mim list` just lists those packages that they have
```
Comment (ZwwWayne) [inline mim/commands/list.py:62]: only need to list those packages that have

File: mim/commands/list.py [hunk 4]
```diff
@@ -52,7 +52,20 @@ def list_package(all: bool = False) -> List[Tuple[str, ...]]:
...
+                # model-index.yml or model_zoo.yml file
+                # more datails at https://github.com/open-mmlab/mim/issues/71
+                installed_path = get_installed_path(pkg_name)
+            except Exception:
```
Comment (ZwwWayne) [inline mim/commands/list.py:66]: file.

File: mim/utils/utils.py [hunk 5]
```diff
@@ -79,12 +79,12 @@ def wrapper(package):
     return wrapper
 
 
-@ensure_installation
+# @ensure_installation
```
Comment (ZwwWayne) [inline mim/utils/utils.py:82]: May delete this line rather than comment it.

Comment (zhouzaida) [reply mim/utils/utils.py]: `ensure_installation` is required and I forgot to remove the `#`.

**Pull request merged**